pkg/ipcache/gc.go

```diff
@@ -58,6 +67,16 @@ func newAsyncPrefixReleaser(parent prefixReleaser, interval time.Duration) *asyn
 func (pr *asyncPrefixReleaser) enqueue(prefixes []netip.Prefix, reason string) {
 	pr.Lock()
 	defer pr.Unlock()
+	select {
+	case <-pr.closeChan:
```

Comment:
Can we block on the `result.Trigger channel` here?

Reply:
We can't block directly on the result's trigger channel because it's not exposed. I've done the next best thing, PTAL.